Index: ruoyi-admin/src/main/resources/application-druid.yaml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- ruoyi-admin/src/main/resources/application-druid.yaml	(revision 9342721e874e7c702226380a387722e06ccfff41)
+++ ruoyi-admin/src/main/resources/application-druid.yaml	(date 1586872571328)
@@ -79,27 +79,28 @@
   # 微信支付地址
   wxPayUrl: http://yuluncc.top/pay
 
+demo:
+  service:
+    version: 1.0.0
+
+embedded:
+  zookeeper:
+    port: 2181
+
+
 dubbo:
   registry:
-    address: 127.0.0.1:2181
-    protocol: zookeeper
-    timeout: 15000
+    address: zookeeper://127.0.0.1:2181
   protocol:
-    port: 12345
     name: dubbo
-    status: server
-    serialization: fastjson
+    port: -1
+    serialization: hessian2
     threads: 2000
-  application:
-    name: dubbo-sms-zk-service
-    id: dubbo-sms-zk-service
   scan:
-    basePackages: com.ruoyi.sms.facade,com.ruoyi.system.facade
-    qos:
-      enable: true
-      port: 44444
-      foreign:
-        ip: false
+    base-packages: com.ruoyi.sms.facade,com.ruoyi.system.facade
+  provider:
+    retries: 3
+    time-out: 10000
 
 
 nacos:
Index: ruoyi-service/src/main/java/com/ruoyi/sms/facade/UserDetailFacadeImpl.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- ruoyi-service/src/main/java/com/ruoyi/sms/facade/UserDetailFacadeImpl.java	(revision 9342721e874e7c702226380a387722e06ccfff41)
+++ ruoyi-service/src/main/java/com/ruoyi/sms/facade/UserDetailFacadeImpl.java	(date 1586869346989)
@@ -5,7 +5,6 @@
 package com.ruoyi.sms.facade;
 
 
-import com.alibaba.dubbo.config.annotation.Service;
 import com.ruoyi.sms.domain.TUserDetail;
 import com.ruoyi.sms.domain.TWechatAuth;
 import com.ruoyi.sms.facade.dto.TWechatAuthDTO;
@@ -14,6 +13,7 @@
 import com.ruoyi.sms.facade.request.UserWechatLoginRequest;
 import com.ruoyi.sms.repository.UserDetailRepository;
 import com.ruoyi.sms.repository.WechatAuthRepository;
+import org.apache.dubbo.config.annotation.Service;
 import org.springframework.beans.factory.annotation.Autowired;
 
 import java.util.UUID;
Index: ruoyi-mp/config/application-dev.yml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- ruoyi-mp/config/application-dev.yml	(revision 9342721e874e7c702226380a387722e06ccfff41)
+++ ruoyi-mp/config/application-dev.yml	(date 1586869190082)
@@ -37,45 +37,19 @@
     name: ruoyi-mp
 
 
+demo:
+  service:
+    version: 1.0.0
 
-endpoints:
-  dubbo:
-    enabled: true
+embedded:
+  zookeeper:
+    port: 2181
 
-
-management:
-  server:
-    port: 8082
-  endpoint:
-    dubbo-configs:
-      enabled: true
-    dubbo:
-      enabled: true
-    dubbo-services:
-      enabled: true
-    dubbo-properties:
-      enabled: true
-    dubbo-shutdown:
-      enabled: true
-    dubbo-references:
-      enabled: true
-  health:
-    dubbo:
-      status:
-        defaults: memory
 dubbo:
   registry:
-    address: 127.0.0.1:2181
-    protocol: zookeeper
-    timeout: 15000
-  protocol:
-    port: 12345
-    name: dubbo
-  application:
-    name: ruoyi-mp
-    id: ruoyi-mp
-    qos:
-      enable: true
-      port: 33333
-      foreign:
-        ip: false
\ No newline at end of file
+    address: zookeeper://127.0.0.1:${embedded.zookeeper.port}
+  consumer:
+    version: 1.0.0
+    check: false
+    timeout: 50000
+    validation: true
\ No newline at end of file
Index: ruoyi-service/pom.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- ruoyi-service/pom.xml	(revision 9342721e874e7c702226380a387722e06ccfff41)
+++ ruoyi-service/pom.xml	(date 1586871540984)
@@ -33,17 +33,22 @@
             <artifactId>sms-facade</artifactId>
         </dependency>
 
+
+        <!-- Dubbo Spring Boot Starter -->
+        <dependency>
+            <groupId>org.apache.dubbo</groupId>
+            <artifactId>dubbo-spring-boot-starter</artifactId>
+            <version>2.7.6</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.dubbo</groupId>
+            <artifactId>dubbo</artifactId>
+        </dependency>
         <!-- ZK -->
         <dependency>
             <groupId>org.apache.zookeeper</groupId>
             <artifactId>zookeeper</artifactId>
         </dependency>
-
-        <dependency>
-            <groupId>com.101tec</groupId>
-            <artifactId>zkclient</artifactId>
-        </dependency>
-
         <dependency>
             <groupId>org.apache.curator</groupId>
             <artifactId>curator-framework</artifactId>
@@ -54,23 +59,17 @@
                 </exclusion>
             </exclusions>
         </dependency>
-
-
         <dependency>
-            <groupId>com.alibaba.boot</groupId>
-            <artifactId>dubbo-spring-boot-starter</artifactId>
-            <exclusions>
-                <exclusion>
-                    <artifactId>guava</artifactId>
-                    <groupId>com.google.guava</groupId>
-                </exclusion>
-                <exclusion>
-                    <artifactId>zookeeper</artifactId>
-                    <groupId>org.apache.zookeeper</groupId>
-                </exclusion>
-            </exclusions>
+            <groupId>org.apache.curator</groupId>
+            <artifactId>curator-recipes</artifactId>
+        </dependency>
+        <!-- Spring Boot dependencies -->
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter</artifactId>
         </dependency>
 
+
     </dependencies>
 
 
@@ -164,10 +163,9 @@
                     </dependency>
                 </dependencies>
             </plugin>
-    
+
         </plugins>
     </build>
 
 
-
 </project>
\ No newline at end of file
Index: ruoyi-system/src/main/java/com/ruoyi/system/facade/AccountFacadeImpl.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- ruoyi-system/src/main/java/com/ruoyi/system/facade/AccountFacadeImpl.java	(revision 9342721e874e7c702226380a387722e06ccfff41)
+++ ruoyi-system/src/main/java/com/ruoyi/system/facade/AccountFacadeImpl.java	(date 1586869419596)
@@ -1,7 +1,6 @@
 package com.ruoyi.system.facade;
 
 
-import com.alibaba.dubbo.config.annotation.Service;
 import com.ruoyi.sms.facade.IAccountFacade;
 import com.ruoyi.sms.facade.ISysOrderFacade;
 import com.ruoyi.sms.facade.dto.SysOrderDTO;
@@ -9,6 +8,8 @@
 import com.ruoyi.system.biz.OrderStatusDispatcher;
 import com.ruoyi.system.biz.UserOrderStatusProcessor;
 import com.ruoyi.system.mapper.AccountMapper;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.dubbo.config.annotation.Service;
 import org.near.toolkit.common.EnumUtil;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.transaction.annotation.Transactional;
@@ -28,6 +29,8 @@
         version = "1.0.0",
         timeout = 15000
 )
+@Slf4j
+
 public class AccountFacadeImpl implements IAccountFacade {
     @Autowired
     private AccountMapper accountMapper;
Index: ruoyi-common/pom.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- ruoyi-common/pom.xml	(revision 9342721e874e7c702226380a387722e06ccfff41)
+++ ruoyi-common/pom.xml	(date 1586869089122)
@@ -136,8 +136,6 @@
         </dependency>
 
 
-
-
     </dependencies>
 
 </project>
\ No newline at end of file
Index: ruoyi-service/src/main/java/com/ruoyi/sms/facade/TsFacadeImpl.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- ruoyi-service/src/main/java/com/ruoyi/sms/facade/TsFacadeImpl.java	(revision 9342721e874e7c702226380a387722e06ccfff41)
+++ ruoyi-service/src/main/java/com/ruoyi/sms/facade/TsFacadeImpl.java	(date 1586869052523)
@@ -1,13 +1,13 @@
 package com.ruoyi.sms.facade;
 
 
-import com.alibaba.dubbo.config.annotation.Service;
 import com.ruoyi.common.core.text.Convert;
-import com.ruoyi.sms.mapper.ExtTsMapper;
-import com.ruoyi.sms.mapper.TsMapper;
 import com.ruoyi.sms.domain.Ts;
 import com.ruoyi.sms.domain.TsExample;
 import com.ruoyi.sms.facade.dto.TsDTO;
+import com.ruoyi.sms.mapper.ExtTsMapper;
+import com.ruoyi.sms.mapper.TsMapper;
+import org.apache.dubbo.config.annotation.Service;
 import org.springframework.beans.BeanUtils;
 import org.springframework.beans.factory.annotation.Autowired;
 
Index: ruoyi-service/src/main/java/com/ruoyi/sms/facade/ShipinFacadeImpl.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- ruoyi-service/src/main/java/com/ruoyi/sms/facade/ShipinFacadeImpl.java	(revision 9342721e874e7c702226380a387722e06ccfff41)
+++ ruoyi-service/src/main/java/com/ruoyi/sms/facade/ShipinFacadeImpl.java	(date 1586869357239)
@@ -1,6 +1,5 @@
 package com.ruoyi.sms.facade;
 
-import com.alibaba.dubbo.config.annotation.Service;
 import com.google.common.collect.Lists;
 import com.ruoyi.common.core.text.Convert;
 import com.ruoyi.sms.convert.ShipinConvert;
@@ -10,6 +9,7 @@
 import com.ruoyi.sms.mapper.ExtShipinMapper;
 import com.ruoyi.sms.mapper.ShipinMapper;
 import com.ruoyi.sms.repository.ShipinRepository;
+import org.apache.dubbo.config.annotation.Service;
 import org.near.servicesupport.result.ResultBuilder;
 import org.near.servicesupport.result.TPageResult;
 import org.near.toolkit.common.StringUtil;
Index: ruoyi-system/src/main/java/com/ruoyi/system/facade/SysWechatConfigFacadeImpl.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- ruoyi-system/src/main/java/com/ruoyi/system/facade/SysWechatConfigFacadeImpl.java	(revision 9342721e874e7c702226380a387722e06ccfff41)
+++ ruoyi-system/src/main/java/com/ruoyi/system/facade/SysWechatConfigFacadeImpl.java	(date 1586869419600)
@@ -1,14 +1,12 @@
 package com.ruoyi.system.facade;
 
-import com.alibaba.dubbo.config.annotation.Service;
 import com.google.common.collect.Lists;
 import com.ruoyi.sms.facade.SysWechatConfigFacade;
-import com.ruoyi.sms.facade.dto.SysWebMainDTO;
 import com.ruoyi.sms.facade.dto.SysWechatConfigDTO;
-import com.ruoyi.system.domain.SysWebMain;
 import com.ruoyi.system.domain.SysWechatConfig;
 import com.ruoyi.system.mapper.SysWechatConfigMapper;
-import com.ruoyi.system.service.ISysWebMainService;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.dubbo.config.annotation.Service;
 import org.springframework.beans.BeanUtils;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.util.CollectionUtils;
@@ -20,6 +18,7 @@
         version = "1.0.0",
         timeout = 15000
 )
+@Slf4j
 public class SysWechatConfigFacadeImpl implements SysWechatConfigFacade {
 
 
Index: ruoyi-service/src/main/java/com/ruoyi/sms/facade/DefaultDemoFacadeImpl.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- ruoyi-service/src/main/java/com/ruoyi/sms/facade/DefaultDemoFacadeImpl.java	(revision 9342721e874e7c702226380a387722e06ccfff41)
+++ ruoyi-service/src/main/java/com/ruoyi/sms/facade/DefaultDemoFacadeImpl.java	(date 1586869363750)
@@ -17,9 +17,9 @@
 package com.ruoyi.sms.facade;
 
 
-import com.alibaba.dubbo.config.annotation.Service;
 import com.google.common.base.Function;
 import com.google.common.collect.Lists;
+import org.apache.dubbo.config.annotation.Service;
 import org.checkerframework.checker.nullness.qual.Nullable;
 
 import java.util.List;
Index: ruoyi-system/src/main/java/com/ruoyi/system/facade/ISysWebMainFacadeImpl.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- ruoyi-system/src/main/java/com/ruoyi/system/facade/ISysWebMainFacadeImpl.java	(revision 9342721e874e7c702226380a387722e06ccfff41)
+++ ruoyi-system/src/main/java/com/ruoyi/system/facade/ISysWebMainFacadeImpl.java	(date 1586869419604)
@@ -1,11 +1,12 @@
 package com.ruoyi.system.facade;
 
-import com.alibaba.dubbo.config.annotation.Service;
 import com.google.common.collect.Lists;
 import com.ruoyi.sms.facade.ISysWebMainFacade;
 import com.ruoyi.sms.facade.dto.SysWebMainDTO;
 import com.ruoyi.system.domain.SysWebMain;
 import com.ruoyi.system.service.ISysWebMainService;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.dubbo.config.annotation.Service;
 import org.springframework.beans.BeanUtils;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.util.CollectionUtils;
@@ -21,6 +22,8 @@
         version = "1.0.0",
         timeout = 15000
 )
+@Slf4j
+
 public class ISysWebMainFacadeImpl implements ISysWebMainFacade {
     @Autowired
     ISysWebMainService sysWebMainService;
Index: pom.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- pom.xml	(revision 9342721e874e7c702226380a387722e06ccfff41)
+++ pom.xml	(date 1586872263836)
@@ -31,11 +31,9 @@
         <poi.version>3.17</poi.version>
         <velocity.version>1.7</velocity.version>
 
+        <dubbo.version>2.7.6</dubbo.version>
+        <spring-boot.version>2.2.6.RELEASE</spring-boot.version>
 
-        <dubbo-spring-boot-starter.version>0.2.0</dubbo-spring-boot-starter.version>
-        <zkclient.version>0.2</zkclient.version>
-        <zookeeper.version>[3.4.14,)</zookeeper.version>
-        <curator-framework.version>2.12.0</curator-framework.version>
         <nacos-config-spring-boot.version>0.2.1</nacos-config-spring-boot.version>
 
 
@@ -46,13 +44,8 @@
         <dependencies>
 
             <!-- SpringBoot的依赖配置-->
-            <dependency>
-                <groupId>org.springframework.boot</groupId>
-                <artifactId>spring-boot-dependencies</artifactId>
-                <version>2.1.1.RELEASE</version>
-                <type>pom</type>
-                <scope>import</scope>
-            </dependency>
+
+
 
             <!--阿里数据库连接池 -->
             <dependency>
@@ -235,22 +228,30 @@
                 <artifactId>ruoyi-mp</artifactId>
                 <version>${ruoyi.version</version>
             </dependency>
+
+
+            <!-- Spring Boot -->
             <dependency>
-                <groupId>com.alibaba.boot</groupId>
-                <artifactId>dubbo-spring-boot-starter</artifactId>
-                <version>${dubbo-spring-boot-starter.version}</version>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-dependencies</artifactId>
+                <version>${spring-boot.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
             </dependency>
+            <!-- Apache Dubbo  -->
 
-
-            <!-- ZK -->
             <dependency>
-                <groupId>org.apache.zookeeper</groupId>
-                <artifactId>zookeeper</artifactId>
-                <version>${zookeeper.version}</version>
+                <groupId>org.apache.dubbo</groupId>
+                <artifactId>dubbo</artifactId>
+                <version>${dubbo.version}</version>
                 <exclusions>
                     <exclusion>
-                        <groupId>org.slf4j</groupId>
-                        <artifactId>slf4j-log4j12</artifactId>
+                        <groupId>org.springframework</groupId>
+                        <artifactId>spring</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>javax.servlet</groupId>
+                        <artifactId>servlet-api</artifactId>
                     </exclusion>
                     <exclusion>
                         <groupId>log4j</groupId>
@@ -259,32 +260,26 @@
                 </exclusions>
             </dependency>
 
+            <!-- Zookeeper -->
+
             <dependency>
-                <groupId>com.101tec</groupId>
-                <artifactId>zkclient</artifactId>
-                <version>${zkclient.version}</version>
-                <exclusions>
-                    <exclusion>
-                        <artifactId>slf4j-api</artifactId>
-                        <groupId>org.slf4j</groupId>
-                    </exclusion>
-                    <exclusion>
-                        <artifactId>log4j</artifactId>
-                        <groupId>log4j</groupId>
-                    </exclusion>
-                    <exclusion>
-                        <artifactId>slf4j-log4j12</artifactId>
-                        <groupId>org.slf4j</groupId>
-                    </exclusion>
-                </exclusions>
+                <groupId>org.apache.zookeeper</groupId>
+                <artifactId>zookeeper</artifactId>
+                <version>3.4.13</version>
             </dependency>
 
             <dependency>
                 <groupId>org.apache.curator</groupId>
                 <artifactId>curator-framework</artifactId>
-                <version>${curator-framework.version}</version>
+                <version>4.0.1</version>
             </dependency>
 
+            <dependency>
+                <groupId>org.apache.curator</groupId>
+                <artifactId>curator-recipes</artifactId>
+                <version>4.0.1</version>
+            </dependency>
+
 
             <dependency>
                 <groupId>cn.hutool</groupId>
@@ -292,7 +287,6 @@
                 <version>5.2.4</version>
             </dependency>
 
-            <!-- https://mvnrepository.com/artifact/com.google.code.gson/gson -->
             <dependency>
                 <groupId>com.google.code.gson</groupId>
                 <artifactId>gson</artifactId>
Index: ruoyi-admin/src/main/resources/application.yaml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- ruoyi-admin/src/main/resources/application.yaml	(revision 9342721e874e7c702226380a387722e06ccfff41)
+++ ruoyi-admin/src/main/resources/application.yaml	(date 1586870264247)
@@ -115,26 +115,7 @@
   # 匹配链接
   urlPatterns: /system/*,/monitor/*,/tool/*
 
-
-management:
-  endpoint:
-    dubbo-configs:
-      enabled: true
-    dubbo:
-      enabled: true
-    dubbo-services:
-      enabled: true
-    dubbo-properties:
-      enabled: true
-    dubbo-shutdown:
-      enabled: true
-    dubbo-references:
-      enabled: true
-  health:
-    dubbo:
-      status:
-        defaults: memory
-        extras: load,threadpool
+debug: true
 
 sequence:
   workerId: 1
\ No newline at end of file
Index: ruoyi-service/src/main/java/com/ruoyi/sms/facade/EnterpriseInfoFacadeImpl.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- ruoyi-service/src/main/java/com/ruoyi/sms/facade/EnterpriseInfoFacadeImpl.java	(revision 9342721e874e7c702226380a387722e06ccfff41)
+++ ruoyi-service/src/main/java/com/ruoyi/sms/facade/EnterpriseInfoFacadeImpl.java	(date 1586869360188)
@@ -1,12 +1,12 @@
 package com.ruoyi.sms.facade;
 
 
-import com.alibaba.dubbo.config.annotation.Service;
 import com.ruoyi.common.core.text.Convert;
 import com.ruoyi.sms.domain.EnterpriseInfo;
 import com.ruoyi.sms.facade.dto.EnterpriseInfoDTO;
 import com.ruoyi.sms.mapper.EnterpriseInfoMapper;
 import com.ruoyi.sms.mapper.ExtEnterpriseInfoMapper;
+import org.apache.dubbo.config.annotation.Service;
 import org.springframework.beans.BeanUtils;
 import org.springframework.beans.factory.annotation.Autowired;
 
Index: ruoyi-mp/pom.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- ruoyi-mp/pom.xml	(revision 9342721e874e7c702226380a387722e06ccfff41)
+++ ruoyi-mp/pom.xml	(date 1586871263065)
@@ -25,14 +25,22 @@
         </dependency>
 
         <!-- ZK -->
+
+
+        <!-- Dubbo Spring Boot Starter -->
         <dependency>
-            <groupId>org.apache.zookeeper</groupId>
-            <artifactId>zookeeper</artifactId>
+            <groupId>org.apache.dubbo</groupId>
+            <artifactId>dubbo-spring-boot-starter</artifactId>
+            <version>2.7.6</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.dubbo</groupId>
+            <artifactId>dubbo</artifactId>
         </dependency>
 
         <dependency>
-            <groupId>com.101tec</groupId>
-            <artifactId>zkclient</artifactId>
+            <groupId>org.apache.zookeeper</groupId>
+            <artifactId>zookeeper</artifactId>
         </dependency>
 
         <dependency>
@@ -48,18 +56,8 @@
 
 
         <dependency>
-            <groupId>com.alibaba.boot</groupId>
-            <artifactId>dubbo-spring-boot-starter</artifactId>
-            <exclusions>
-                <exclusion>
-                    <artifactId>guava</artifactId>
-                    <groupId>com.google.guava</groupId>
-                </exclusion>
-                <exclusion>
-                    <artifactId>zookeeper</artifactId>
-                    <groupId>org.apache.zookeeper</groupId>
-                </exclusion>
-            </exclusions>
+            <groupId>org.apache.curator</groupId>
+            <artifactId>curator-recipes</artifactId>
         </dependency>
 
 
Index: ruoyi-system/src/main/java/com/ruoyi/system/facade/ISysOrderFacadeImpl.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- ruoyi-system/src/main/java/com/ruoyi/system/facade/ISysOrderFacadeImpl.java	(revision 9342721e874e7c702226380a387722e06ccfff41)
+++ ruoyi-system/src/main/java/com/ruoyi/system/facade/ISysOrderFacadeImpl.java	(date 1586869390650)
@@ -1,6 +1,5 @@
 package com.ruoyi.system.facade;
 
-import com.alibaba.dubbo.config.annotation.Service;
 import com.google.common.collect.Lists;
 import com.ruoyi.sms.facade.ISysOrderFacade;
 import com.ruoyi.sms.facade.dto.SysOrderDTO;
@@ -10,6 +9,7 @@
 import com.ruoyi.system.mapper.ExtSysOrderMapper;
 import com.ruoyi.system.service.ISysOrderService;
 import lombok.extern.slf4j.Slf4j;
+import org.apache.dubbo.config.annotation.Service;
 import org.near.toolkit.common.StringUtil;
 import org.springframework.beans.BeanUtils;
 import org.springframework.beans.factory.annotation.Autowired;
Index: ruoyi-system/pom.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- ruoyi-system/pom.xml	(revision 9342721e874e7c702226380a387722e06ccfff41)
+++ ruoyi-system/pom.xml	(date 1586871242425)
@@ -36,16 +36,21 @@
         </dependency>
 
 
+        <!-- Dubbo Spring Boot Starter -->
+        <dependency>
+            <groupId>org.apache.dubbo</groupId>
+            <artifactId>dubbo-spring-boot-starter</artifactId>
+            <version>2.7.6</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.dubbo</groupId>
+            <artifactId>dubbo</artifactId>
+        </dependency>
         <!-- ZK -->
         <dependency>
             <groupId>org.apache.zookeeper</groupId>
             <artifactId>zookeeper</artifactId>
         </dependency>
-
-        <dependency>
-            <groupId>com.101tec</groupId>
-            <artifactId>zkclient</artifactId>
-        </dependency>
 
         <dependency>
             <groupId>org.apache.curator</groupId>
@@ -58,23 +63,15 @@
             </exclusions>
         </dependency>
 
-
         <dependency>
-            <groupId>com.alibaba.boot</groupId>
-            <artifactId>dubbo-spring-boot-starter</artifactId>
-            <exclusions>
-                <exclusion>
-                    <artifactId>guava</artifactId>
-                    <groupId>com.google.guava</groupId>
-                </exclusion>
-                <exclusion>
-                    <artifactId>zookeeper</artifactId>
-                    <groupId>org.apache.zookeeper</groupId>
-                </exclusion>
-            </exclusions>
+            <groupId>org.apache.curator</groupId>
+            <artifactId>curator-recipes</artifactId>
         </dependency>
 
 
+
+
+
         <dependency>
             <groupId>com.alibaba.boot</groupId>
             <artifactId>nacos-config-spring-boot-starter</artifactId>
Index: ruoyi-admin/src/test/resources/application.yaml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- ruoyi-admin/src/test/resources/application.yaml	(revision 9342721e874e7c702226380a387722e06ccfff41)
+++ ruoyi-admin/src/test/resources/application.yaml	(date 1586868506416)
@@ -116,25 +116,6 @@
   urlPatterns: /system/*,/monitor/*,/tool/*
 
 
-management:
-  endpoint:
-    dubbo-configs:
-      enabled: true
-    dubbo:
-      enabled: true
-    dubbo-services:
-      enabled: true
-    dubbo-properties:
-      enabled: true
-    dubbo-shutdown:
-      enabled: true
-    dubbo-references:
-      enabled: true
-  health:
-    dubbo:
-      status:
-        defaults: memory
-        extras: load,threadpool
 
 sequence:
   workerId: 1
\ No newline at end of file
Index: ruoyi-admin/src/test/resources/application-dev.yaml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- ruoyi-admin/src/test/resources/application-dev.yaml	(revision 9342721e874e7c702226380a387722e06ccfff41)
+++ ruoyi-admin/src/test/resources/application-dev.yaml	(date 1586869783172)
@@ -78,27 +78,29 @@
   wxPayUrl: http://yuluncc.top/pay
 
 
+
+demo:
+  service:
+    version: 1.0.0
+
+embedded:
+  zookeeper:
+    port: 2181
+
 dubbo:
   registry:
-    address: 127.0.0.1:2181
-    protocol: zookeeper
-    timeout: 15000
+    address: zookeeper://127.0.0.1:${embedded.zookeeper.port}
   protocol:
-    port: 12345
     name: dubbo
-    status: server
-    serialization: fastjson
+    port: -1
+    serialization: hessian2
     threads: 2000
-  application:
-    name: dubbo-sms-zk-service
-    id: dubbo-sms-zk-service
   scan:
-    basePackages: com.ruoyi.sms.facade,com.ruoyi.system.facade
-    qos:
-      enable: true
-      port: 44444
-      foreign:
-        ip: false
+    base-packages: com.ruoyi.sms.facade,com.ruoyi.system.facade
+  provider:
+    retries: 3
+    time-out: 10000
+
 
 nacos:
   config:
Index: ruoyi-admin/src/test/resources/application-druid.yaml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- ruoyi-admin/src/test/resources/application-druid.yaml	(revision 9342721e874e7c702226380a387722e06ccfff41)
+++ ruoyi-admin/src/test/resources/application-druid.yaml	(date 1586869783169)
@@ -79,27 +79,28 @@
   # 微信支付地址
   wxPayUrl: http://yuluncc.top/pay
 
+demo:
+  service:
+    version: 1.0.0
+
+embedded:
+  zookeeper:
+    port: 2181
+
+
 dubbo:
   registry:
-    address: 127.0.0.1:2181
-    protocol: zookeeper
-    timeout: 15000
+    address: zookeeper://127.0.0.1:${embedded.zookeeper.port}
   protocol:
-    port: 12345
     name: dubbo
-    status: server
-    serialization: fastjson
+    port: -1
+    serialization: hessian2
     threads: 2000
-  application:
-    name: dubbo-sms-zk-service
-    id: dubbo-sms-zk-service
   scan:
-    basePackages: com.ruoyi.sms.facade,com.ruoyi.system.facade
-    qos:
-      enable: true
-      port: 44444
-      foreign:
-        ip: false
+    base-packages: com.ruoyi.sms.facade,com.ruoyi.system.facade
+  provider:
+    retries: 3
+    time-out: 10000
 
 
 nacos:
Index: ruoyi-admin/src/main/resources/logback.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- ruoyi-admin/src/main/resources/logback.xml	(revision 9342721e874e7c702226380a387722e06ccfff41)
+++ ruoyi-admin/src/main/resources/logback.xml	(date 1586870094617)
@@ -76,7 +76,7 @@
 	<!-- Spring日志级别控制  -->
 	<logger name="org.springframework" level="warn" />
 
-	<root level="info">
+	<root level="debug">
 		<appender-ref ref="console" />
 	</root>
 	
Index: ruoyi-mp/config/application-druid.yml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- ruoyi-mp/config/application-druid.yml	(revision 9342721e874e7c702226380a387722e06ccfff41)
+++ ruoyi-mp/config/application-druid.yml	(date 1586869215739)
@@ -39,44 +39,19 @@
 
 
 
-endpoints:
-  dubbo:
-    enabled: true
+demo:
+  service:
+    version: 1.0.0
 
+embedded:
+  zookeeper:
+    port: 2181
 
-management:
-  server:
-    port: 8082
-  endpoint:
-    dubbo-configs:
-      enabled: true
-    dubbo:
-      enabled: true
-    dubbo-services:
-      enabled: true
-    dubbo-properties:
-      enabled: true
-    dubbo-shutdown:
-      enabled: true
-    dubbo-references:
-      enabled: true
-  health:
-    dubbo:
-      status:
-        defaults: memory
 dubbo:
   registry:
-    address: 127.0.0.1:2181
-    protocol: zookeeper
-    timeout: 15000
-  protocol:
-    port: 12345
-    name: dubbo
-  application:
-    name: ruoyi-mp
-    id: ruoyi-mp
-    qos:
-      enable: true
-      port: 33333
-      foreign:
-        ip: false
\ No newline at end of file
+    address: zookeeper://127.0.0.1:${embedded.zookeeper.port}
+  consumer:
+    version: 1.0.0
+    check: false
+    timeout: 50000
+    validation: true
\ No newline at end of file
Index: ruoyi-admin/src/main/resources/application-dev.yaml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- ruoyi-admin/src/main/resources/application-dev.yaml	(revision 9342721e874e7c702226380a387722e06ccfff41)
+++ ruoyi-admin/src/main/resources/application-dev.yaml	(date 1586872564158)
@@ -78,27 +78,29 @@
   wxPayUrl: http://yuluncc.top/pay
 
 
+
+demo:
+  service:
+    version: 1.0.0
+
+embedded:
+  zookeeper:
+    port: 2181
+
 dubbo:
   registry:
-    address: 127.0.0.1:2181
-    protocol: zookeeper
-    timeout: 15000
+    address: zookeeper://127.0.0.1:2181
   protocol:
-    port: 12345
     name: dubbo
-    status: server
-    serialization: fastjson
+    port: -1
+    serialization: hessian2
     threads: 2000
-  application:
-    name: dubbo-sms-zk-service
-    id: dubbo-sms-zk-service
   scan:
-    basePackages: com.ruoyi.sms.facade,com.ruoyi.system.facade
-    qos:
-      enable: true
-      port: 44444
-      foreign:
-        ip: false
+    base-packages: com.ruoyi.sms.facade,com.ruoyi.system.facade
+  provider:
+    retries: 3
+    time-out: 10000
+
 
 nacos:
   config:
